resources/assets/less/store/supporter-tag.less

```diff
@@ -0,0 +1,138 @@
+.store-supporter-tag {
+  &__slider {
+    .ui-slider {
+      background-color: darkgray;
+      .ui-slider-range {
+        background-color: @pink-darker;
+      }
+    }
+  }
+
+  &__user-icon {
+    margin: 8px;
+  }
+
+  &__user-search {
+    margin: 8px;
+
+    .grid-cell {
+      margin: 0 0 8px 0;
+    }
+
+    input.form-control {
+      background-color: #eee;
+      text-align: center;
+      min-width: 14em;
+    }
+  }
+
+  &--disabled {
+    .js-slider-preset {
```

Comment:
not sure about styling `js-` prefixed class in its own context (not as modifier)